woocommerce-addressfinder.php

```diff
@@ -70,7 +70,7 @@ function add_addressfinder_widget( $_checkout ) {
 
 	// Add the tab to the tabs array
 	function filter_addressfinder_settings_tabs_array( $settings_tabs ) {
-    $settings_tabs['addressfinder-settings'] = __( 'Addressfinder Settings', 'woocommerce' );
```

Comment:
I thought settings was a good label but thats just me 😆

Reply:
Yep same, but Matt is right there, these tabs are within the Settings page, so it makes little sense having "Settings" there